src/components/buoy-telemetry/BuoyMap.vue

```diff
@@ -111,10 +265,36 @@ onMounted(() => {
       layout: {
         "icon-allow-overlap": true,
         "icon-image": "buoy-marker",
-        "icon-offset": [0.5, -7], // optically centered
+        "icon-size": 1.2,
+        "icon-offset": [0.8, -4], // optically centered
       },
     });
 
+    for (var b = 0; b < windSpeedBins.length - 1; b++) {
+      map.value.addLayer({
+        id: "svgfile" + b,
+        type: "symbol",
+        filter: [
+          "all",
+          [">=", "windSpeed", windSpeedBins[b]],
+          ["<", "windSpeed", windSpeedBins[b + 1]],
+        ],
+        source: "points", // points.filter(annotatedwind)?
```

Comment:
taking out commented code
```suggestion
        source: "points",
```

Reply:
deleted!